src/packages/preprocessing.py

```
@@ -0,0 +1,60 @@
+    def __init__(self, filename, config) -> None:
+
+        self.config = config
+        self.all_paths = AllPaths(config= self.config)
+        self.df_input = self.csv_loader_raw_data()
+        self.filename = filename
+        self.X = None
+        self.y = None
+        self.preprocessor = None
+
+    def csv_loader_raw_data(self, filename):
+        return spark.read.option("header",True).csv(f'{self.all_paths.data_volume}/{filename}.csv').toPandas()
```

Ruff

Undefined name `spark`

(F821)